### DISCUSSION TITLE
YASI

### DISCUSSION THREAD

**Discussion by yasamanrasteh** [WeAreMahsaAmini/Core#56] [Introductions]
👋 Hi everyone, I'm Yasi, a Graphic, and UX/UI Designer. 
I am great at Illustrator/Photoshop/Figma. 
I know a bit Html/Css, if it could be useful. 
I work Mon-Fri/ until 5 pm (Toronto). However, I am willing to spend the rest on IRAN.

**Comment by syavash** [comment]
Welcome, @yasamanrasteh. We were actually talking about a project that requires illustration. Please join us on Discord: https://discord.com/invite/ASDSFjGAAW
